test/geojson2h3.spec.js

```diff
@@ -134,7 +134,7 @@ function assertSymmetrical(assert, feature, hexagons) {
     );
     assertEqualFeatures(
         assert,
-        h3SetToFeature(featureToH3Set(feature, DEFAULT_RES)),
+        h3SetToFeature(featureToH3Set(feature, DEFAULT_RES).sort()),
```

Comment:
Why this change?

Reply:
The test was a little brittle. `h3-js@3.7.1` changes the output order of `polyfill` results, and this test depended on the order. Sorting everything normalizes.